.github/workflows/ci.yml

```diff
@@ -62,22 +65,32 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npm install
-      - run: npm install sequelize@${{ matrix.sequelize-version }}
-      - run: npm run build
-      - run: npm run test
+      - run: yarn install --frozen-lockfile --ignore-engines
+      - run: yarn add sequelize@${{ matrix.sequelize-version }}
+      - run: yarn test
   release:
     name: Release
     runs-on: ubuntu-latest
-    needs: [test-sqlite, test-postgres, test-mysql]
-    if: github.event_name == 'push' && github.ref == 'refs/heads/release'
+    needs:
+      [
+        test-sqlite,
+        test-postgres,
+        test-mysql
+      ]
+    if: github.event_name == 'push' && (github.ref == 'refs/heads/v6' || github.ref == 'refs/heads/v7')
     env:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/setup-node@v2
         with:
-          node-version: 12.x
-      - run: npm install
+          node-version: 16.x
+      - run: yarn install --frozen-lockfile
       - run: npx semantic-release
+      - id: sequelize
+        uses: sdepold/github-action-get-latest-release@master
+        with:
+          repository: sequelize/cli
+      - run: |
+          curl -XPOST -u "sdepold:${{ secrets.GH_TOKEN }}" -H "Accept: application/vnd.github.v3+json" -H "Content-Type: application/json" https://api.github.com/repos/sequelize/cli/dispatches --data '{"event_type":"Release notifier","client_payload":{"release-id": ${{ steps.sequelize.outputs.id }}}}'
```

Comment:
@sdepold two/three questions about this;
1. Is this something we want for CLI as well?
2. Why does it refer to secrets.GH_TOKEN here instead of secrets.GITHUB_TOKEN? (the same is in the core repo, I just copied that from here)
3. If the answer to 1. is positive, the answer to 2. makes us require both; is secrets.GH_TOKEN already set up? If not, could you do that?

Reply:
:wave:

This command is triggering the following flow (we'll have to copy the file over): https://github.com/sequelize/sequelize/blob/main/.github/workflows/notify.yml

The prime reason for this line is:

1. I wanted to decouple the notification part from the release pipeline. 
2. I originally observed published releases in the notification pipeline but it turned out that the pipeline won't trigger if the release happened by some other pipeline in the first place.
3. To circumvent the problem, you can dispatch an event from one pipeline, however that (according to my studies on SO and the web) has to happen from another users "session" (in lack of a better word). So the auth tokens have to differ from the one that is running the original pipeline.
4. In short: The ci.yml pipeline does all the things to ensure that tests are green and that releases happen. It will then send a dispatch event using my (or any other) account to the same repo. In the notify.yml pipeline, we listen for such dispatch events and will notify all the relevant channels (currently Twitter and OpenCollective) about the newly made release.

I think it would be nice to have the notify pipeline in place and I can add the GH_TOKEN secret

Reply:
I just added the secret

Reply:
I've added the notify pipeline as well, guess you'll need to add those secrets as well